spec/features/summary_feature_spec.rb

```diff
@@ -0,0 +1,76 @@
+require 'rails_helper'
+
+feature 'sub tabs' do
```

Comment:
I think this feature spec could benefit from having a more descriptive name.

If this tests the tab navigation on the male prisoners summary page, then something along the lines of this might be more descriptive:

```suggestion
feature 'male prisoners summary navigation tabs' do
```